services/app-api/src/resolvers/healthPlanPackage/indexHealthPlanPackages.ts

```diff
@@ -52,7 +53,7 @@ export function indexHealthPlanPackagesResolver(
                 user.stateCode
             )
             return validateAndReturnHealthPlanPackages(results, span)
-        } else if (isCMSUser(user)) {
+        } else if (isCMSUser(user) || isAdminUser(user)) {
             const results = await store.findAllHealthPlanPackagesBySubmittedAt()
 
```

Comment:
@rswerve Just a heads up (In case you haven't run into this), after allowing Admin users to fetch a single health plan the summary page works. Going to an unlocked submission leads to a 404 page. I encountered this on a local deployment.